extensions/wikia/NjordPrototype/css/Njord.scss

```diff
@@ -261,14 +261,21 @@
 	}
 
 	.title-wrap {
-		margin: 30px 330px 20px 30px;
+		margin: 0;
```

Comment:
yes we do, unless we do refactoring of this stylesheet which I'd like to do in the next iteration of MoM project